en/mkdocs.yml

```diff
@@ -268,9 +268,20 @@ nav:
     - 'EMail OTP Configurations': guides/mfa/email-otp-config-advanced.md
     - 'SMS OTP Configurations': guides/mfa/sms-otp-config-advanced.md
     - 'TOTP Configurations': guides/mfa/totp-config-advanced.md
+  - 'Admin Portal':
+      - 'Applications':
+          - admin-portal/applications/overview.md
```

Comment:
With this 'Applications' heading will be duplicated in the site. Better to have a sensible heading.